tests/pages/firefox/do_not_track.py

```diff
@@ -31,15 +31,23 @@ class FrequentlyAskedQuestion(PageRegion):
 
     _question_locator = (By.CSS_SELECTOR, 'h3[role="tab"]')
     _answer_locator = (By.CSS_SELECTOR, 'div[role="tabpanel"]')
+    _expanded_locator = (By.CSS_SELECTOR, 'h3[aria-expanded="true"]')
 
     def show_answer(self):
         assert not self.is_answer_displayed, 'Answer is already displayed'
-        el = self.root.find_element(*self._answer_locator)
-        self.root.find_element(*self._question_locator).click()
-        Wait(self.selenium, self.timeout).until(expected.visibility_of(el))
+        question = self.root.find_element(*self._question_locator)
+        answer = self.root.find_element(*self._answer_locator)
+        # Prevent Selenium automatically scrolling this element to the top of
+        # the page, which can lead to unintented clicks on the sticky header
+        self.selenium.execute_script('arguments[0].scrollIntoView(false);', question)
```

Comment:
I'm not convinced we need this force scroll into view. I think the issue is much more likely to be the element presenting as visible before it's finished transitioning.